examples/webpack.config.js

```diff
@@ -21,6 +21,12 @@ module.exports = {
       test: /\.js$/,
       loaders: ['react-hot', 'babel'],
       include: path.join(__dirname, 'src')
+    }, {
+      test: /\.js$/,
+      loaders: ['react-hot', 'babel'],
+      include: path.join(__dirname, '..', 'src')
```

Comment:
I added these lines above b/c they existed on my react-json-tree repo but I forgot to copy them over:
https://github.com/chibicode/react-json-tree/blob/d37ecea16027dae64bda561332a5bbce00587886/examples/webpack.config.js#L36-L40